app/shared/recurring-reservation-controls/_recurring-reservation-controls.scss

```diff
@@ -6,4 +6,23 @@
     white-space: nowrap;
     overflow: hidden;
   }
+
+  .date-picker {
+    height: $input-height-base;
+    display: flex;
+    justify-items: center;
+    flex-direction: column;
+    border: none;
+
+    input {
```

Comment:
Is there a way to attach a className instead for targeting the element's name?

Reply:
Under this component there are only 1 date-picker then i dont think its necessary